package.json

```diff
@@ -38,26 +38,24 @@
     "node": ">=0.8"
   },
   "dependencies": {
-    "assert-plus": "0.1.2",
-    "bignum": "0.6.0",
-    "bunyan": "*",
+    "assert-plus": "1.0.0",
+    "bignum": "0.13.0",
+    "bunyan": "2.0.2",
```

Comment:
Latest bunyan 1.x please. 2.x is still "beta/experimental".

Reply:
will do